WordPress/src/main/res/values/strings.xml

```diff
@@ -1513,6 +1513,12 @@
     <string name="my_site_btn_plugins">Plugins</string>
     <string name="my_site_create_new_site">Create a new site for your business, magazine, or personal blog; or connect an existing WordPress installation.</string>
     <string name="my_site_add_new_site">Add new site</string>
+    <string name="my_site_icon_dialog_title">Site Icon</string>
+    <string name="my_site_icon_dialog_add_message">There is no icon set. Would you like to add one?</string>
+    <string name="my_site_icon_dialog_change_message">How would you like to change the icon?</string>
```

Comment:
Have we asked Editorial about this copy?  The `my_site_icon_dialog_add_message` and `my_site_icon_dialog_change_message` strings may need to be reviewed if not.

Reply:
Hi @theck13, thanks for the ping. In the first instance, I'd like to suggest a small trim: 

Original: 
> There is no icon set. Would you like to add one?

Proposed: 
> Would you like to add a site icon?

On the second string, what are the choices available to the user in this context: "How would you like to change the icon?"

Reply:
> Hi @theck13, thanks for the ping.

No problem.  Thanks for reviewing!

> Would you like to add a site icon?

That sounds good to me.

> On the second string, what are the choices available to the user in this context: "How would you like to change the icon?"

The options are "Remove" and "Change" in that case.  See the screenshot below for illustration.

<img src="https://user-images.githubusercontent.com/3827611/39942703-8be0f776-551d-11e8-93f0-b75dd95b508c.png" width="240">

Reply:
Ah, thank you! How about: 

> Would you like to edit the icon? 

And then the options could be: 

> DELETE    |    ADD NEW

Reply:
> Would you like to edit the icon?

That question seems like it's asking for a Yes / No answer rather than Add New / Delete.  In that case, what if I didn't want to add a new one or delete the existing one?  Maybe we should add a third button as shown in the screenshot below so users can dismiss the dialog without changing the icon.  Users can tap the back button to dismiss the dialog, but having an explicit action as another dismissal option would be better.

<img src="https://user-images.githubusercontent.com/3827611/39944416-dd71c5ce-5523-11e8-876f-fb175f3861fb.png" width="240">

Reply:
For the initial question, I'd suggest: 

> How would you like to edit the icon? 

I like the idea of adding CANCEL -- that definitely improves the user experience.  It seems odd to me that we're using REMOVE and CHANGE here as options. I know we use that same text in Calypso though it seems strange, considering that we use ADD and DELETE or TRASH as conventions for media and posts and pages. The ultimate decision is yours, though I think we should try to be consistent wherever possible to help users learn by convention instead of using different terms to mean the same thing. I realize that I may be missing a compelling reason for this inconsistency.

Reply:
> How would you like to edit the icon?

That sounds good to me.

> I think we should try to be consistent wherever possible to help users learn by convention instead of using different terms to mean the same thing.

I totally agree.  That second screenshot was just meant to show where the third button would be located and not suggest what the copy should be.  I guess Change / Remove do make sense since that's what Calypso uses and that would help with consistency for this specific feature.

Thanks again for the review, @kristastevens!  If you feel it's okay with the changes we discussed, can you remove the `[Status] Needs Copy Review`?

Reply:
It seems I'm not able to remove the label, though I consider this request closed. :)